.github/workflows/e2e_hypernode.yaml

```diff
@@ -1,11 +1,7 @@
 name: E2E Hypernode
 
 on:
-  push:
-    branches:
-      - master
-    tags:
-  pull_request:
+  workflow_call:
```

Comment:
Sorry I'm not so familiar with workflow_call, so all the CIs can stil be triggered throught the e2e.yaml, right?

Reply:
<img width="970" height="407" alt="image" src="https://github.com/user-attachments/assets/517172c6-ea3f-4149-a154-fe437cefb002" />
But I found that we still have these old pending checks waiting to be reported, but we already have execute these CIs, we should not execute them twice, right?